src/server/routes/github.js

```diff
@@ -10,5 +10,6 @@ router.post('/github/webhook', createWebhook);
 
 router.use('/github/repos', json());
 router.get('/github/repos', listRepositories);
+router.get('/github/repos/page/:page', listRepositories);
```

Comment:
Not a blocker, but to me the page feels more like a query string parameter than a route parameter.  (That would also match GitHub.)

Whichever way you go, this could also do with corresponding changes to the route documentation in `docs/routes.md`.